stories/API.stories.tsx

```diff
@@ -519,6 +519,38 @@ export const DeleteSheet: ComponentStory<typeof Workbook> = () => {
   );
 };
 
+export const UpdateSheet: ComponentStory<typeof Workbook> = () => {
+  const ref = useRef<WorkbookInstance>(null);
+  const [data, setData] = useState<Sheet[]>([
+    { id: "1", name: "sheet1", data: [[{ v: "1" }]], order: 0 },
+  ]);
+  const onChange = useCallback((d: Sheet[]) => {
+    setData(d);
+  }, []);
+  return (
+    <ApiExecContainer
+      onRun={() => {
+        ref.current?.updateSheet([
```

Comment:
Can we have similar method to get the current sheet data as well before updating

Reply:
You can call getSheet in api
<img width="728" alt="image" src="https://user-images.githubusercontent.com/84694078/209684358-326fd78c-9113-4aa4-85b2-8f65a5eb5ab5.png">
https://ruilisi.github.io/fortune-sheet-docs/guide/api.html#getsheet